src/applications/disability-benefits/2346/components/UnverifiedPrefillAlert.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleLoginModal } from 'platform/site-wide/user-nav/actions';
+
+const UnverifiedPrefillAlert = props => (
+  <div className="usa-alert usa-alert-info schemaform-sip-alert">
```

Comment:
Just to confirm but can we combine these divs (and their classes) since it only has 1 child and it's also a div?

Reply:
tried this and it seems like the previous div is flexed to row and the body is flexed column so they need to remain nested

Reply:
Ah perfect, makes sense!